### ISSUE TITLE
Wrap py3status modules

### ISSUE THREAD

**Issue by ramnes** [qtile/qtile#940]
[py3status](https://github.com/ultrabug/py3status) has [numerous cool modules](https://github.com/ultrabug/py3status/tree/master/py3status/modules) that we could use easily by creating a wrapper widget that would bind our widgets implementation methods to py3status corresponding modules methods.

I'm just leaving this here to share the idea and propose it if someone else want to give it a shot.
Since I'm used to both implementations, and thinking about doing this for some time now, I might tackle it one day, but I think it could be a great project for a new contributor that want to discover Qtile widgets. :hatching_chick:

**Comment by ultrabug** [comment]
There's no politics in OSS as far as I'm concerned and I'd be more than happy to hear that the py3status project benefits users of qtile !

Silly idea maybe as I have no clue of what that implies but **maybe py3status could be made to natively be used as the source of the status bar of qtile** ?

I guess if qtile has something like the i3bar protocol we would just have to change py3status' output format to match qtile's bar input format ;)

**Comment by ramnes** [comment]
@tobes

**Comment by jwijenbergh** [comment]
py3status is still maintained after all these years which is great. However, do we still want this?

**Comment by ramnes** [comment]
Yeah I guess so? Or maybe that can be a package that lives outside Qtile if someone wants to tackle this
